### PR TITLE
chore: removes unused imports, updates others

### PR DESCRIPTION
**Pull request by noahamac**
Before, *4.83mb*
image

Now, *2.77mb*
image

Seems good to me, but I suspect we can go further? Do you have any ideas about further optimization? 

Will remove the webpack bundle analyzer before merging.

### CODE REVIEW

File: package.json
Comment (noahamac) [inline package.json:3]: Is this the magic word?

File: webpack.config.js
Comment (noahamac) [inline webpack.config.js:3]: Will remove before merging. Is helpful

File: package.json
Comment (bryans99) [reply package.json:3]: The css certainly needs to be included in the side effects but there is a bunch of other stuff that has been done to reduce the module size. It's much better but can be improved more I believe. This is good improvement though.
What I would really like to see however is a general cleanup of how CSS is used. Imports of CSS should be replaced with styled components

File: webpack.config.js
Comment (bryans99) [reply webpack.config.js:3]: Leave in. See below

File: webpack.config.js
Comment (bryans99) [inline webpack.config.js:33]: This is how it's done in the examples
```
    new BundleAnalyzerPlugin({
      analyzerMode: process.env.ANALYZE_MODE || 'disabled',
    }),
```
In `package.json` scripts
```
 "analyze": "export ANALYZE_MODE=static && yarn build",
 ```
Not sure you need the stats file.

File: src/components/ExternalLink.tsx
Comment (bryans99) [inline src/components/ExternalLink.tsx:35]: Ugh!
Try this as an alternative (I have not tested but if you are in here let's clean this up
```
import React, {useContext} from "react"
import { Link, LinkProps } from "@looker/components"
import { Looker40SDK } from '@looker/sdk'
import {
  ExtensionContext2,
  ExtensionContextData2,
} from '@looker/extension-sdk-react'

export const ExternalLink: React.FC<Omit<LinkProps, "color">> = (
  {href, target, onClick, ...rest}
) => {
  const extensionContext = useContext<ExtensionContextData2<Looker40SDK>>(
    ExtensionContext2
  )
  const { extensionSDK } = extensionContext
  const onLinkClick = (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
    // probably don't need to call this as href is no longer added to the Link
    event.stopPropagation()
    if (href) {
      extensionSDK.updateLocation(
        href,
        undefined,
        target
      )
    } else if (onClick) {
      onClick(event)
    }
  }

  return (
    <Link onClick={onLinkClick} {...rest} />
  )
}
```

**Review by bryans99 — COMMENTED** [review]
Good improvement but we can do more. For another PR though.

See suggestions for some changes. 

Will approve if you disagree.

Note new suggested ExternalLink implementation has not been tested. Thinking about it, is it actually needed? Can you just use looker components Link component directly? If does not seem to buy you much.

File: src/components/ExternalLink.tsx
Comment (noahamac) [reply src/components/ExternalLink.tsx:35]: Clutch code snippet! This works for me. I stole the previous one from DD

File: webpack.config.js
Comment (noahamac) [reply webpack.config.js:3]: Nice tip

File: package.json
Comment (bryans99) [reply package.json:3]: So I saw a separate PR to get rid of the css. At the end of the day sideEffects has an impact on modules but this is not a module. Most likely delete for now

**Review by bryans99 — APPROVED** [review]
I dont think you need the sideEffects. Aside from that its okay

**Pull request merged**
